data/quarg outfits.txt

```diff
@@ -50,6 +51,7 @@ outfit "Quarg Skylance"
 		"shield damage" 50
 		"hull damage" 40
 		"ion damage" .1
+	description "The skylance is seemingly built from a smooth, black, granite-like material, although obviously it is actually made of a far more advanced and sophisticated substance."
```

Comment:
What *I* was hoping to use as a good description for the skylance went like this:
`The immensely powerful beam weapons used by Quarg warships strike fear into the hearts of all who dare oppose them.`

Reply:
i just used something you posted as a place holder till something better was made